### PR TITLE
GRW-404 / Feature / Enable new embark story routes + update landing page design

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Add SE needer home + accident to landing page.
Add SE switcher (without accident) to landing page.
Enable routes in production.
Update design of landing page to match updates from @zakkjellin.
Refactor landing page components to use standard breakpoints.
Update text to standard typography.
Update translations.

## Why?

@gustaveen has created new (and translated) Angel stories for SE with support for Accident that we need to link to from the landing page. We start with home + accident needer and home switcher.

New copy has been updated for the landing page so we need to tweak the design to match updates in Figma.

_Referenced ticket(s): [GRW-404]_

## Demo

Mobile

Screenshot 2021-11-01 at 21 41 16

Tablet

Screenshot 2021-11-01 at 21 41 39

Desktop (en)

Screenshot 2021-11-01 at 21 41 58

Desktop (DK, example of another market)

Screenshot 2021-11-01 at 21 42 18

[GRW-404]: https://hedvig.atlassian.net/browse/GRW-404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mehdielket — DISMISSED** [review]
Nice with the breakpoint cleanup! Well done :)

**Review by guilhermespopolin — DISMISSED** [review]

File: src/client/pages/Landing/landingPageData.ts [hunk 1]
```diff
@@ -29,13 +29,13 @@ export const productsData: ProductsData = {
   SE: [
     {
       id: 'swedishNew',
-      linkSlug: '/new',
+      linkSlug: '/home-accident-needer',
```
Comment (gustaveen) [inline src/client/pages/Landing/landingPageData.ts:32]: I'm not sure we should change the routes though. I'm thinking in terms of tracking and if we links to the flows directly. But we can check with Sonny I guess

**Comment by gustaveen** [comment]
Great Robin! I'm wondering if we should enable the routes in production already though. Merging this would block us from releasing to production.

**Comment by gustaveen** [comment]
So I've updated with a temporary text key not mentioning Accident so we can merge to production (to get the icon fix out). https://github.com/HedvigInsurance/web-onboarding/pull/650
Screenshot 2021-11-02 at 15 07 22
Screenshot 2021-11-02 at 15 07 28

File: src/client/pages/Landing/landingPageData.ts (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Landing/landingPageData.ts]: Aha, very true!

**Review by robinandeer — COMMENTED** [review]
Looks good to me @gustaveen!

I also updated the color of the paragraph text based on Zak's input

File: src/client/pages/Landing/components/Card.tsx [hunk 2]
```diff
@@ -137,8 +137,8 @@ export const CardParagraph = styled.p`
   padding-right: 2rem;
   font-size: 1rem;
   line-height: 1.5;
-  color: ${colorsV3.gray700};
-  opacity: 0.76;
+  /* Gray/@gray600 */
```
Comment (gustaveen) [inline src/client/pages/Landing/components/Card.tsx:140]: 👍

**Review by gustaveen — APPROVED** [review]
We can merge this IMO! @robinandeer 🙌 Create a review app if anyone else wanna check the landing page :)

**Pull request merged**